### ISSUE TITLE
Turn on security for testing

### ISSUE THREAD

**Issue by ruflin** [elastic/beats#27211]
Currently for testing against Elatsicsearch snapshots, security is turned off by default: https://github.com/elastic/beats/blob/master/testing/environments/snapshot.yml#L16 As Elasticsearch is enabling security by default in 8.0 we should also do this for our testing to uncover potential issues early on.

**Comment by cachedout** [comment]
@ruflin Could you please have a look at this PR and see if it's what you had in mind? https://github.com/elastic/beats/pull/28131

**Comment by cachedout** [comment]
@jlind23 I don't think we can close it until https://github.com/elastic/beats/pull/28131 is complete.

**Comment by jlind23** [comment]
Sorry didn't see it 🤦🏼‍♂️

**Comment by cachedout** [comment]
Status update: just waiting for final reviews on https://github.com/elastic/beats/pull/28131 and once that is merged we can close this.
